querydsl-jdo/README.md

```diff
@@ -71,9 +71,10 @@ Querying with Querydsl JDO is as simple as this :
 ```JAVA
 QCustomer customer = QCustomer.customer;
 JDOQuery query = new JDOQuery(pm);
```

Comment:
Better? :wink: 
I thought I had them all, good that you also spotted that one. :+1: